docs/resources/subaccounts.md

```diff
@@ -1,46 +1,24 @@
 # Subaccounts
 
-This library provides easy access to the [Subaccounts](https://www.sparkpost.com/api#/reference/subaccounts) Resource.
+This library provides easy access to the [Subaccounts](https://developers.sparkpost.com/api/subaccounts) Resource.
+
+*Note: All methods return promises and accept an optional last argument callback. [Read about how we handle callbacks and promises](/docs/async.md).*
 
 ## Methods
-* **all(callback)**
+* **list([callback]) &rarr; `{Promise}`**<br />
```

Comment:
I was getting rid of both the `[, callback]` and the `&rarr; {Promise}` stuff in these lines because I like how the note at the top just covers all of the async cases for everything at once, then the docs are de-cluttered to just show the things that are important for that individual method.